machine-learning/configs.py

```diff
@@ -87,12 +89,22 @@
 
 # Empty dataset means 'all datasets'
 # options = ['', 'apache', 'github', 'fdroid']
-DATASETS = ['', 'apache', 'github', 'fdroid']
+DATASETS = ['']
+
 
 # --------------------------------
 # Refactorings
 # --------------------------------
 
+#refactoring levels
+class Level(IntEnum):
+    NONE = 0
+    Class = 1
+    Method = 2
+    Variable = 3
+    Field = 4
+    Other = 5
```

Comment:
Do we need this 'other' ? Are we going to train on those?

Reply:
Probably, the two relevant refactorings are: "Move Source Folder" and "Change Package". I would still include them in the general config and remove them if we want to.